forge/db/views/ProjectSnapshot.js

```diff
@@ -55,6 +55,22 @@ module.exports = function (app) {
             return null
         }
     }
+    app.addSchema({
+        $id: 'ExportedSnapshot',
+        type: 'object',
+        properties: {
+            id: { type: 'string' },
+            name: { type: 'string' },
+            description: { type: 'string' },
+            createdAt: { type: 'string' },
+            updatedAt: { type: 'string' },
+            user: { $ref: 'UserSummary' },
+            userWhoExported: { $ref: 'UserSummary' },
```

Comment:
```suggestion
            exportedBy: { $ref: 'UserSummary' },
```

I think this is a more consistent property name. Note I have only proposed this change here - you'll need to modify the other places in this PR that uses this property name, including tests.

Reply:
Done :)